package.json

```diff
@@ -132,10 +132,21 @@
           "type": "boolean",
           "default": false
         },
-        "sway-lsp.debug.showParsedTokensAsWarnings": {
-          "markdownDescription": "Show successfully parsed tokens by the sway-lsp server as warnings. If set to false, sway-lsp will revert to only showing warnings and errors reported by the compiler.",
-          "default": false,
-          "type": "boolean"
+        "sway-lsp.debug.showCollectedTokensAsWarnings": {
+          "scope": "window",
+          "type": "string",
+          "description": "Show either successfully parsed or typed tokens by the sway-lsp server as warnings. If set to off, sway-lsp will revert to only showing warnings and errors reported by the compiler.",
```

Comment:
Should there be an option to see both parsed and typed?

Reply:
Yeah possibly, that could be useful. We would need another way apart from the squiggly lines to highlight tokens. Perhaps warning lines for the parsed tokens, and forcing the typed tokens to temporarily all have the same semantic color would work. Will have more of a think.